.gitbook/0-about-panacea/1-data-exchange-protocol/2-How-DEP-works.md

```diff
@@ -0,0 +1,104 @@
+# How DEP Works
+
+In this guide, you can learn about the DEP flow and how DEP works.
+
+- [DEP flow](#dep-flow)
+
+You can also see how we solved the problem described on [DEP and the problems it solves](./1-DEP-problems-it-solves.md);
+
+- [For data privacy](#for-data-privacy)
+- [For data validation](#for-data-validation)
+- [Atomic incentive](#atomic-incentive-distribution)
+- [Decentralization & Scalability](#decentralization--scalability)
+
+## DEP flow
+
+![](images/user-flow.drawio.png)
+
+The diagram describes each key player interacts with each other to complete a cycle of data exchange. The whole process consists of several steps below.
```

Comment:
This DEP component par overlaps with the content of [user-flow](https://github.com/medibloc/panacea-core/blob/main/.gitbook/3-protocol-devs/1-dep-specs/1-user-flow.md). So, I'm thinking about simplifying this part.

Reply:
yeah. we don't need to copy those contents to here. I think it's enough to put a link to the user flow.

Reply:
if you added some new contents in this doc (compared to the user-flow doc), you can move them to the user-flow doc.

Reply:
Thanks, I simplify that part and put a link to `user-flow`